### PR TITLE
Update release instructions

### PR DESCRIPTION
**Pull request by Shabirmean**
### Adds more details to the release instructions

Change summary:
- Update few details related to releasing the repo and deploying 

Remaining issues / concerns:
- NA

### CODE REVIEW

File: release/README.md [hunk 1]
```diff
@@ -59,9 +74,13 @@ This process is going to trigger multiple CI checks as well as stage the release
 ## Add notes to release
 
 Once the PR has been fully merged, you are ready to create a new release for the newly created [tag](https://github.com/GoogleCloudPlatform/bank-of-anthos/tags).
+- Click the breadcrumbs on the row of the latest tag that was created in the [tags](https://github.com/GoogleCloudPlatform/bank-of-anthos/tags) page
+- Select the `Create release` option
 
 The release notes should contain a brief description of the changes since the previous release (like bug fixed and new features). For inspiration, you can look at the list of [releases](https://github.com/GoogleCloudPlatform/bank-of-anthos/releases).
 
+> ***Note:*** No assets needs to be uploaded. They are picked up automatically from the tagged revision
```
Comment (askmeegs) [inline release/README.md:82]: s/needs/need

File: release/README.md [hunk 2]
```diff
@@ -50,6 +52,19 @@ This script does the following:
 5. Uses `skaffold` to build and push new stable release images to `gcr.io/bank-of-anthos`.
 6. Pushes the Git tags and the release branch.
 
+### Troubleshooting script failures
+
+In the event of any of the steps above failing you might have to revert the repository to it's original state. Follow the steps below as required:
```
Comment (askmeegs) [inline release/README.md:57]: s/it's/its

File: release/README.md [hunk 3]
```diff
@@ -50,6 +52,19 @@ This script does the following:
...
+```sh
+# delete the newly created release branch & tag before re-running the script
+git checkout master
+git branch -D release/v$NEW_VERSION
```
Comment (askmeegs) [inline release/README.md:61]: I think you can remove the "v" before `$NEW_VERSION`, in the steps above, `NEW-VERSION` already has a "v"

File: release/README.md [hunk 4]
```diff
@@ -74,7 +93,9 @@ gcloud container clusters get-credentials bank-of-anthos-master --zone us-west1-
 
 Currently the `bank-of-anthos-master` cluster has [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) enabled. Thus, when deploying to this cluster the pod service account (`boa-ksa-master`) used by _Workload Identity_ must be used as the serviceAccount in the manifests.
 
-Follow the steps 3 and 4 of the the [workload identity setup](https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/master/docs/workload-identity.md) with `boa-ksa-master` as the `KSA_NAME` to deploy into production.
+Follow the steps 3 and 4 of the the [workload identity setup](https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/master/docs/workload-identity.md) with the following config values to deploy into production:
```
Comment (askmeegs) [inline release/README.md:96]: extra "the"

**Review by askmeegs — CHANGES_REQUESTED** [review]

File: release/README.md (hunk 4 above)
Comment (Shabirmean) [reply release/README.md]: Fixed in 7bb9544363ac55987a066e55c24a0fff2b5ab3b6

File: release/README.md (hunk 2 above)
Comment (Shabirmean) [reply release/README.md]: Fixed in 7bb9544363ac55987a066e55c24a0fff2b5ab3b6

File: release/README.md (hunk 1 above)
Comment (Shabirmean) [reply release/README.md]: Fixed in 7bb9544363ac55987a066e55c24a0fff2b5ab3b6

File: release/README.md (hunk 3 above)
Comment (Shabirmean) [reply release/README.md]: Fixed in 7bb9544363ac55987a066e55c24a0fff2b5ab3b6

**Review by askmeegs — APPROVED** [review]

**Pull request merged**